click_odoo_contrib/backupdb.py

```diff
@@ -11,11 +11,25 @@
 import click
 import click_odoo
 from click_odoo import odoo
+from decorator import decorator
 
 from ._backup import backup
 from ._dbutils import db_exists
 
 
+def db_management_enabled(method):
+    def force_db_management_enabled(method, self, *args, **kwargs):
+        list_db = odoo.tools.config["list_db"]
+        odoo.tools.config["list_db"] = True
+        try:
+            return method(self, *args, **kwargs)
+        finally:
+            odoo.tools.config["list_db"] = list_db
+
+    return decorator(force_db_management_enabled, method)
```

Comment:
a context manager would be a tad more flexible (we could use it in [other contexts](https://github.com/acsone/click-odoo-contrib/commit/826f663d100156b883b19d33ca11adfed4e7eefd)).

Reply:
OK code adapted